src/ng-app/assets/locale/lang.nl.xlf

```diff
@@ -0,0 +1,1403 @@
+          <context context-type="linenumber">69</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="cryptPasswordInputCurrentCrypt">
+        <source> Please enter your current crypt password </source>
+        <target phase-name="approval-1" state="signed-off"> Voer het huidige kluiswachtwoord in </target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/change-crypt-password/change-crypt-password.component.html</context>
+          <context context-type="linenumber">44,45</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="cryptPasswordInputNewCrypt">
+        <source> Please enter your new crypt password </source>
+        <target phase-name="approval-1" state="signed-off"> Voer het nieuwe kluiswachtwoord in </target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/change-crypt-password/change-crypt-password.component.html</context>
+          <context context-type="linenumber">53,54</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="cryptPasswordNotMatch">
+        <source> Passwords do not match </source>
+        <target phase-name="approval-1" state="signed-off"> De wachtwoorden komen niet overeen </target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/change-crypt-password/change-crypt-password.component.html</context>
+          <context context-type="linenumber">62,63</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="cProfMgrDetailsFanChartCPULabel">
+        <source>CPU Fan</source>
+        <target phase-name="approval-1" state="signed-off">Cpu-ventilator</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/fan-graph/fan-graph.component.ts</context>
+          <context context-type="linenumber">64</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="cProfMgrDetailsFanChartGPULabel">
+        <source>GPU Fan</source>
+        <target phase-name="approval-1" state="signed-off">Gpu-ventilator</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/fan-graph/fan-graph.component.ts</context>
+          <context context-type="linenumber">73</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit datatype="html" approved="yes" id="messageCPUSettingsOff">
+        <source>CPU settings deactivated in Tools→Global Settings</source>
+        <target phase-name="approval-1" state="signed-off">De cpu-instellingen zijn uitgeschakeld via Hulpmiddelen → Algemene instellingen</target>
```

Comment:
```suggestion
        <target phase-name="approval-1" state="signed-off">De CPU-instellingen zijn uitgeschakeld via Hulpmiddelen → Algemene instellingen</target>
```

Reply:
Also note, the formatting here is slightly different with spaces surrounding the arrow.

Reply:
Spaces surrounding the arrow is for readability, which is how it's usually done (esp. in Dutch).